joomla/joomla-cms
# [6.0] Responsive pagination

Pull Request for Issue #32674

### Summary of Changes
Uses css media query etc to provide responsive pagination

Made for 5.2 as I have no idea if this is considered a bug fix or a feature - can easily be rebased

When you have a lot of items in a list (site and admin) then it breaks the layout on small devices or requires horizontal scrolling

### Testing Instructions
As this is a scss change either use a pre-built package from this PR or apply the pr and run `npm run build:css`

Create a list view in both the site and admin and check the pagination


### Actual result BEFORE applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/1296369/f3bbaff0-6f0b-4780-90b5-20390f01a894)



### Expected result AFTER applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/1296369/1ead03fb-ce33-4af1-92a9-69d559762cc1)

### Additional Checks
Make sure that the arrows are always displayed and that the ellipsis is never displayed between the arrows

![image](https://github.com/joomla/joomla-cms/assets/1296369/49a7952f-3be7-4b93-8807-cbf6c19c4480)



### Link to documentations
Please select:
